v2/server.go

```diff
@@ -237,6 +243,9 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// Prevents Internet Explorer from MIME-sniffing a response away
 	// from the declared content-type
 	w.Header().Set("x-content-type-options", "nosniff")
+	if s.allowCORS {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
This can be extremely insecure as a default, since it allows any cross-origin request. I don’t think gorilla/rpc is the place to support CORS.